be/src/storage/lake/tablet_manager.cpp

```diff
@@ -10,49 +19,123 @@
 
 namespace starrocks::lake {
 
+static const char* const kStarletPrefix = "staros_";
+
 TabletManager::TabletManager(Starlet* starlet, int64_t cache_capacity)
         : _starlet(starlet), _metacache(new_lru_cache(cache_capacity)) {}
 
+// path $bucket:/$ServiceID/$GroupID/tbl_${TabletID}_${version}
+std::string TabletManager::tablet_meta_path(const std::string& group, int64_t tablet_id, int64_t verson) {
+    return fmt::format("{}{}/tbl_{}_{}", kStarletPrefix, group, tablet_id, verson);
+}
+
+// trnslog path rule $Bucket:/$ServiceID/$GroupID/txn_${TabletID}_${TxnID}
+std::string TabletManager::txn_log_path(const std::string& group, int64_t tablet_id, int64_t txn_id) {
+    return fmt::format("{}{}/txn_{}_{}", kStarletPrefix, group, tablet_id, txn_id);
+}
+
 Status TabletManager::create_tablet(const TCreateTabletReq& req) {
+#ifdef USE_STAROS
+    // generate metapb
+    TabletMetadataPB tablet_metadata_pb;
+
+    // get shard info
+    ASSIGN_OR_RETURN(staros::starlet::ShardInfo shard_info, get_shard_info(req.tablet_id));
+
+    // write tablet meta
+    auto meta_path =
+            tablet_meta_path(shard_info.obj_store_info.uri, tablet_metadata_pb.id(), tablet_metadata_pb.version());
+    ASSIGN_OR_RETURN(auto fs, FileSystem::CreateSharedFromString(meta_path));
+    ASSIGN_OR_RETURN(auto wf, fs->new_writable_file(meta_path));
+    RETURN_IF_ERROR(wf->append(tablet_metadata_pb.SerializeAsString()));
+    RETURN_IF_ERROR(wf->close());
+    return Status::OK();
+#else
     (void)req;
     return Status::NotSupported("TabletManager::create_tablet");
+#endif
 }
 
 StatusOr<Tablet> TabletManager::get_tablet(int64_t tablet_id) {
+#ifdef USE_STAROS
+    ASSIGN_OR_RETURN(staros::starlet::ShardInfo shard_info, get_shard_info(tablet_id));
+    return Tablet(this, shard_info.obj_store_info.uri, tablet_id);
+#else
     (void)tablet_id;
     return Status::NotSupported("TabletManager::get_tablet");
+#endif
 }
 
+// called when user drop the entire table,remove all related tablet_meta,and trn_log;
+// tablet meta rule $Bucket:/$ServiceID/$GroupID/tbl_${TabletID}_${version}
+// trnslog path rule $Bucket:/$ServiceID/$GroupID/txn_${TabletID}_${TxnID}
+// data files  $Bucket:/${ServiceID}/${GroupID}/UUID.dat not deleted
 Status TabletManager::drop_tablet(int64_t tablet_id) {
+#ifdef USE_STAROS
+    ASSIGN_OR_RETURN(staros::starlet::ShardInfo shard_info, get_shard_info(tablet_id));
+    auto tablet_meta_prefix = fmt::format("{}/tbl_{}_", shard_info.obj_store_info.uri, tablet_id);
+    auto trns_log_prefix = fmt::format("{}/txn_{}_", shard_info.obj_store_info.uri, tablet_id);
+    auto object_store = _starlet->get_store(tablet_meta_prefix);
+    if (object_store == nullptr) {
+        return Status::InternalError(fmt::format("failed to get object store from starlet {}", tablet_meta_prefix));
+    }
+    auto st = object_store->delete_object(tablet_meta_prefix);
```

Comment:
`delete_object()` is used for deleting a single object, not deleting objects with the specified prefix.